iam-migrate.go

```diff
@@ -46,10 +52,10 @@ func (adm *AdminClient) ExportIAM(ctx context.Context) (io.ReadCloser, error) {
 }
 
 // ImportIAM makes an admin call to setup IAM  from imported content
-func (adm *AdminClient) ImportIAM(ctx context.Context, contentReader io.ReadCloser) error {
+func (adm *AdminClient) ImportIAM(ctx context.Context, contentReader io.ReadCloser) (sam SkippedIAMEntities, err error) {
```

Comment:
```suggestion
func (adm *AdminClient) ImportIAM(ctx context.Context, contentReader io.ReadCloser) (sam SkippedIAMEntities, err error) {
```

Never break existing APIs, introduce new APIs.

Reply:
ImportIAMV2

Reply:
I am doing this second time, that's a sin ;) Will correct and will keep note of this.